### PR TITLE
PMM-10176 Create Release Notes for PMM 2.29.0

### PR DESCRIPTION
**Pull request by rasika-chivate**

### CODE REVIEW

File: docs/release-notes/2.29.0.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+ - Upgraded VictoriaMetrics operatot to version 0.24.
+ - Upgraded the ClickHouse plugin to 2.4.4 which fixes some of the CVEs.
+ - Added RHEL 9 support for PMM client.
+ - With this release, we are upgrading grpc-gateway to version 2.0, which deprecates the error field in error responses. If you are using the PMM API, make sure to replace this with the new message field from the Google Cloud API Error model. 
+
+
+
+### Important bug fixes
+
+- [PMM-9981](https://jira.percona.com/browse/PMM-9981):
```
Comment (catalinaadam) [inline docs/release-notes/2.29.0.md:87]: empty

File: docs/release-notes/2.29.0.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+  
+- Instead of typing an exact filter pattern, you can now intuitively create a filter by choosing from the available operators and filling in the predefined fields. So instead of typing: *service_name=name123*, just fill in the following fields: 
+  ![!filters](../_images/filter_fields.png) 
+  
+- [PMM-9611](https://jira.percona.com/browse/PMM-9611): PMM now warns you if your current domain is different than the one specified in the Grafana .INI configuration file. Since this can generate incorrect short URLs when sharing the link to a PMM dashboard, make sure to correct this mismatch if you see a warning on the **Share Panel > Link page**. 
+  
+- [PMM-7326](https://jira.percona.com/browse/PMM-7326): You now have more control over the logs you want to view with the **--log-level** parameter. With this enhancement, you can experience more pronounced logging with the flexibility to choose options such as warning, error, and fatal.
+  
+- [PMM-8566](https://jira.percona.com/browse/PMM-8566): For enhanced security, you can now specify the custom Basic Auth password for agents when adding Node
```
Comment (catalinaadam) [inline docs/release-notes/2.29.0.md:121]: is this sentence complete?

File: docs/release-notes/2.29.0.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+
+- [PMM-10039](https://jira.percona.com/browse/PMM-10039): VictoriaMetrics operator has been upgraded to 0.24.
+
+- [PMM-10083](https://jira.percona.com/browse/PMM-10083): Upgraded the ClickHouse plugin to 2.4.4 which fixes some of the CVEs.
+
+- [PMM-10103](https://jira.percona.com/browse/PMM-10103): Added RHEL 9 support for PMM client.
+
+- [PMM-10155](https://jira.percona.com/browse/PMM-10155): Access QAN from the main menu instead of the PMM dashboard for better reach and visibility.
+
+- [PMM-2038](https://jira.percona.com/browse/PMM-2038):
```
Comment (catalinaadam) [inline docs/release-notes/2.29.0.md:151]: empty entry

Comment (rasika-chivate) [reply docs/release-notes/2.29.0.md:151]: Waiting for inputs from the developer.

**Comment by rnovikovP** [comment]
:siren:   @rasika-chivate,  @catalinaadam  I see in Release notes  "copy/paste" from my highlights. :scream:   Like "More details and targets in commands."
It was not supposed to go "as is "; it was just a "High-level plan of what and how to group."
I see the "Percona Platform Integration" section as a  good example of presenting data.

File: docs/release-notes/2.29.0.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+
+### Exporters
+
+Updated the MySQL exporter.
+
+### Components upgrade
+
+ - Upgraded VictoriaMetrics operatot to version 0.24.
+ - Upgraded the ClickHouse plugin to 2.4.4 which fixes some of the CVEs.
+ - Added RHEL 9 support for PMM client.
```
Comment (rnovikovP) [inline docs/release-notes/2.29.0.md:80]: this is already mentioned in line 26

File: docs/release-notes/2.29.0.md [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+
+Use the filters on the **Advisors Checks > All Checks** page to search advisors by **Name**, **Description**, **Status** or **Interval**. If you need to share the filtered results with your team members, send them the PMM URL which saves your search criteria and results. 
+
+### DBaaS
+
+Simplified the API such that it requires fewer parameters to run the API. Also, enhanced the documentation.
+
+### Exporters
+
+Updated the MySQL exporter.
```
Comment (rnovikovP) [inline docs/release-notes/2.29.0.md:74]: Exporters are not independent features, so this is part of the "Components update." But we also need to add a value to this action .. what it gives users

File: docs/release-notes/2.29.0.md [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+
+### Monitoring
+
+- Meta metrics: Additional metrics for Mongodb exporters to monitor the time it takes to collect data from different collectors. This allows you to identify the monitoring costs for various collectors by MongoDB exporters.
+- PXC Cluster Dashboard: [Experimental]: Restructured the PXC cluster Dashboard to make it intuitive and user-friendly. Important data for the PXC cluster is put together in this dashboard to obtain feedback from our users.
+
+### Troubleshooting and operations improvement
+
+- Logs verbosity management: Pronounced control over the logs you want to view with the flexibility to choose options such as warning, error, and fatal with the --log-level parameter.
+- More details and targets in commands
```
Comment (rnovikovP) [inline docs/release-notes/2.29.0.md:36]: This was a  placeholder about all small improvement we got for this topic

**Review by rnovikovP — CHANGES_REQUESTED** [review]
Also: images will be better to use in Highlights and have links to documentation in related features. 

Example:  Troubleshooting  improvements - link to the pmm-admin reference for flags update (all new flags, parameters need to be described there )

**Comment by denisok** [comment]
@rasika-chivate you forgot to update indexes and vars. Release notes for 2.29.0 aren't shown in the index/menu

**Review by denisok — CHANGES_REQUESTED** [review]

File: docs/release-notes/2.29.0.md (hunk 4 above)
Comment (rasika-chivate) [reply docs/release-notes/2.29.0.md:168]: > @rasika-chivate you forgot to update indexes and vars. Release notes for 2.29.0 aren't shown in the index/menu

@denisok I will be creating a separate ticket for that will the changes to the index as well as the config file.

**Review by denisok — CHANGES_REQUESTED** [review]
My suggestions still not landed. Please also update index, so it will show up in the menu.

**Review by denisok — APPROVED** [review]

**Review by rnovikovP — APPROVED** [review]

**Pull request merged**